### PR TITLE
bug: 'database is locked' error after some time without calling the API

### PR DESCRIPTION
**Pull request by lucasmenendez**
After a period of inactivity, the API returns an internal server error caused by a database error: `error saving the census on the database error="database is locked"`.

After a few requests with this error, the API starts working without any issue.

At the moment we do not know how much time of inactivity is required to reproduce the error.

Original trace:
```
2023-08-29T10:20:00.095Z ERR api/censuses.go:188 > error saving the census on the database error="database is locked"
2023-08-29T10:20:10.521Z ERR api/censuses.go:188 > error saving the census on the database error="database is locked"
2023-08-29T10:20:17.18Z ERR api/censuses.go:188 > error saving the census on the database error="database is locked"
2023-08-29T10:23:17.608Z ERR api/censuses.go:188 > error saving the census on the database error="database is locked"
```

We found some references about SQLite `database is locked` error on the official documentation of the SQL driver that we use ([mattn/go-sqlite3](https://github.com/mattn/go-sqlite3#faq)) and in its issues https://github.com/mattn/go-sqlite3/issues/274#issuecomment-232942571.

### CODE REVIEW

**Comment by lucasmenendez** [comment]
The implemented fix comes from https://github.com/vocdoni/vocdoni-node/blob/0e0af7447473d98affff2e4ccd2217a45ec95846/vochain/indexer/indexer.go#L120-L143

File: service/holder_scanner_test.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func Test_saveHolders(t *testing.T) {
 	// check update holders
 	th.Append(holderAddr, holderBalance)
 	c.Assert(hs.saveHolders(th), qt.IsNil)
-	res, err = testdb.queries.TokenHolderByTokenIDAndHolderID(context.Background(),
+	res, err = testdb.db.QueriesRW.TokenHolderByTokenIDAndHolderID(context.Background(),
```
Comment (mvdan) [inline service/holder_scanner_test.go:149]: is this RO?

File: service/holder_scanner_test.go [hunk 2]
```diff
@@ -157,7 +157,7 @@ func Test_saveHolders(t *testing.T) {
 	// check delete holders
 	th.Append(holderAddr, big.NewInt(-24))
 	c.Assert(hs.saveHolders(th), qt.IsNil)
-	_, err = testdb.queries.TokenHolderByTokenIDAndHolderID(context.Background(),
+	_, err = testdb.db.QueriesRW.TokenHolderByTokenIDAndHolderID(context.Background(),
```
Comment (mvdan) [inline service/holder_scanner_test.go:160]: is this RO?

File: api/censuses.go [hunk 3]
```diff
@@ -49,7 +49,7 @@ func (capi *census3API) getCensus(msg *api.APIdata, ctx *httprouter.HTTPContext)
 			log.Errorw(err, "holders transaction rollback failed")
 		}
 	}()
-	qtx := capi.sqlc.WithTx(tx)
+	qtx := capi.db.QueriesRO.WithTx(tx)
```
Comment (mvdan) [inline api/censuses.go:52]: FYI you seem to use a transaction for a single query here, so it's unnecessary

**Review by mvdan — APPROVED** [review]
maybe squash the commits, since you ended up just copying what the indexer does

**Pull request merged**